packages/helpers/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Let's keep a single `tsconfig.json` in root folder for all packages.

Reply:
I'll try to move to root folder.

Reply:
Add a global `tsconfig` in root folder, and extends global `tsconfig` in `helpers`.

I don't find the way to solve the `outDir` problem, so a add a `tsconfig` and extends global one.